src/PageView/Login/userData.js

```diff
@@ -0,0 +1,13 @@
+import React, { useState, createContext } from 'react';
+
+export const UserContext = createContext();
+
+export function UserDataProvider(props) {
+    const [user,setUser] = useState([{name:'John Doe', url: 'idk'}]);
+    
+    return (
+        <UserContext.Provider value={[user, setUser]}>
```

Comment:
The context receives an object, not an array as value. by using {} you just tell react that you will be using something else than a string. in your case, I think you should use value={{ user }}

Reply:
Hi, Ian! I tried applying the change, and it's getting an error. I'll leave it as it is for now for the team app review, then I'll make the necessary changes to fix the issue :)